web_app/testSite.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Test</title>
+    <link rel="stylesheet" type="text/css" href="src/form.css">
+</head>
+<body>
+    <div id="task-modal" class="modal">
+        <div class="modal-content">
+        <span class="close">&times;</span>
+        <form id="frame">
+            <h1 id="new-task-form-title">Create new Task</h1>
```

Comment:
I'm not a huge fan of using `<h1>` in places like this, it makes writing our own styling a tad more annoying and messes with screen readers. I'd personally use `<p>` here, but your call